jrnl/cli.py

```diff
@@ -63,7 +63,7 @@ def guess_mode(args, config):
         compose = False
         export = False
         import_ = True
-    elif args.decrypt is not False or args.encrypt is not False or args.export is not False or any((args.short, args.tags, args.edit)):
+    elif args.decrypt is not False or args.encrypt is not False or args.export is not False or args.edit:
```

Comment:
Did you mean to commit this change? This is a change to guess mode, and the original issue was documentation-only.

Reply:
yes @maebert 	pointed this out in [comment](https://github.com/jrnl-org/jrnl/pull/617#pullrequestreview-260446315) of my old PR. That is why I made these changes.

Reply:
Sorry about the back and forth, but I'm still not sure about this line. It looks like it breaks the functionality. Do the tests pass if you take this one out?

Reply:
The "export mode" is a bit misleading. Short or tags mode only get triggered when "export mode" is True, otherwise it will always default to the normal reading mode. So either you need to revert this change or fix the if-else at the bottom of cli.run() here: https://github.com/jrnl-org/jrnl/blob/master/jrnl/cli.py#L248-L260.

The whole cli.py could use a severe revamp/code refactoring, it's quite a mess.